core/src/main/scala/com/rallyhealth/vapors/factfilter/data/Fact.scala

```diff
@@ -10,18 +10,29 @@ sealed abstract class Fact {
   val typeInfo: FactType[Value]
   val value: Value
 
-  override def toString: String = {
-    // TODO: JSON format for value? Show for value?
-    s"Fact(${typeInfo.fullName} = $value)"
+  // TODO: JSON format for value? Show for value?
+  override def toString: String = this match {
+    case derivedFact: DerivedFact =>
```

Comment:
Why are you using reflection here?

Reply:
Good point. It's a `sealed trait`, so I can just override toString in the case classes.